templates/setenv.sh.erb

```diff
@@ -47,7 +47,7 @@ DISABLE_NOTIFICATIONS=" -Datlassian.mail.senddisabled=true -Datlassian.mail.fetc
 # This allows us to actually debug GC related issues by correlating timestamps
 # with other parts of the application logs.
 #-----------------------------------------------------------------------------------
-JVM_EXTRA_ARGS="-XX:+PrintGCDateStamps"
+JVM_EXTRA_ARGS="<%= scope.lookupvar('jira::jvm_extra') %>"
```

Comment:
instead of using the legacy scope.lookupvar, you should be able to directly access the variable with @jvm_extra

Reply:
scope.lookupvar is used everywhere else in these template, so was just attempting to follow current style.